src/MediaManagerBundle/Resources/public/css/mediaManager.css

```diff
@@ -106,7 +106,7 @@
 .snippetMediaManagerBackendModuleStandard .edit-container {
     position: relative;
     padding-left: 10px;
-    min-height: 250px;
+    min-height: 450px;
```

Comment:
With 500px also the upload bar that appears later on will fit.

Reply:
thanks for finding out. changed it.